mod-source-record-manager-server/src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -95,6 +95,7 @@
   private static final AtomicInteger indexer = new AtomicInteger();
   private static final String HOLDINGS_004_TAG_ERROR_MESSAGE =
     "The 004 tag of the Holdings doesn't has a link to the Bibliographic record";
+  static final String RECORD_ID = "recordId";
```

Comment:
Maybe will be better to add modifier?

Reply:
I think package-private is suitable here to make it possible for test to use it and not to expose it globally.